content/strategy-goals/strategy/iam/Q2FY23-OKR.md

```diff
@@ -0,0 +1,47 @@
+# Q2 Objective and Key Results
+
+### Problem Area: Align on Team Vision
+
+- Key Result 1: An updated handbook page for the IAM team including mission, vision, and team working agreements.
+- Key Result 2: A GitHub board outlining the key efforts on our 12-month roadmap validated by customer product gaps.
+
+**Engineering OKR Alignment:** This does not align to engineering OKRs. It is necessary due to the formation of a new team.
+
+### Problem Area: Unifying Feature Permissions
+
+> NOTE: This work has been paused for Q2.
+
+Today, nearly all Sourcegraph features (Code insights, batch changes, search notebooks, etc) have unique, custom permissions models, creating confusion for users and preventing cross-feature use cases from being developed.
+
+- Key Result 1: A detailed product & technical plan for implementing a unified permissions model
+- Key Result 2: Implementation of the MVP for a unified permissions model
+- Key Result 3: Robust documentation for feature teams to adopt this updated permissions mode
+
+Within this area, in Q2 we are not working on:
+
+- Implementing RBAC for administrators
+- Updating features to respect our permissions, this will be owned by each feature team
+
+**Engineering OKR Alignment:** Objective #2: Focus on time-to-value for a single use-case (Fixing Vulnerabilities). With a single permissions model, Sourcegraph feature teams will be able to enable cross-feature experiences for our customers to fix vulnerabilities. For example, setting up a code monitor to ensure a vulnerability is not re-introduced after fixing with a batch change.
+
+### Problem Area: Improve Admin Analytics
+
+_Update_: The MVP of this work is planning to be launched in the July release of Sourcegraph (version 3.42).
```

Comment:
Should we remove "planning to be" from launched?

Reply:
Fixed! Good catch.